.travis.yml

```diff
@@ -2,4 +2,4 @@ before_install: cd server
 language: java
 
 jdk:
-  - oraclejdk8
+  - openjdk8
```

Comment:
I changed to the openjdk as the oracle jdk requires a licence now which caused Travis checks to fail